src/connectors/cloudflare/index.ts

```diff
@@ -9,6 +9,7 @@ import { GQLAssetType } from 'definitions'
 const envPrefix = isProd ? 'prod' : 'non-prod'
 
 const CLOUDFLARE_IMAGES_URL = `https://api.cloudflare.com/client/v4/accounts/${environment.cloudflareAccountId}/images/v1`
+const CLOUDFLARE_IMAGE_ENDPOINT = `https://imagedelivery.net/${environment.cloudflareAccountId}/${envPrefix}`
 
```

Comment:
can use `https://matters.news/cdn-cgi/imagedelivery/{accountId}/` prefix instead? to avoid have to add extra sites in CSP